hedera-node/src/main/java/com/hedera/services/store/contracts/precompile/impl/TokenCreatePrecompile.java

```diff
@@ -209,8 +209,8 @@ public void run(final MessageFrame frame) {
 		final var treasuryId = Id.fromGrpcAccount(tokenCreateOp.getTreasury());
 		final var treasuryHasSigned = KeyActivationUtils.validateKey(
 				frame, treasuryId.asEvmAddress(), sigsVerifier::hasActiveKey, ledgers, updater.aliases());
-		validateTrue(treasuryHasSigned, INVALID_SIGNATURE);
-		tokenCreateOp.getAdminKey().ifPresent(key -> validateTrue(validateAdminKey(frame, key), INVALID_SIGNATURE));
+		validateTrue(treasuryHasSigned, INVALID_FULL_PREFIX_SIGNATURE_FOR_PRECOMPILE, "Invalid full prefix for token create precompile!");
```

Comment:
nit: would be cleaner to make all the "Invalid full prefix ...." static properties in these classes

Reply:
We can have a common property with value "Invalid full prefix for %s precompile!" and different static values for each precompile (`"mint"`, `"burn"`, `"associate"`, etc.) and then do `String.format(INVALID_FULL_PREFIX, MINT)`